src/pages/configuration/index.vue

```diff
@@ -1,7 +1,160 @@
 <template>
-  <div class="w-full" />
+  <div v-if="loading">
+    <Loading />
+  </div>
+  <AppLayout>
+    <template #title>
+      Configuration
+    </template>
+    <template #actions>
+      <AppButton v-if="!editing" @click="startEditing()">
```

Comment:
You can directly use `startEditing` instead of calling it. It is more idiomatic.